File: runtimes/dotnet-6.0/src/RuntimeOutput.cs
namespace DotNetRuntime
{
	public class RuntimeOutput
	{
		public string Body { get; set; }
		public int StatusCode { get; set; }
		public Dictionary<string, string> Headers { get; set; }

		public RuntimeOutput(string Body, int StatusCode, Dictionary<string, string> Headers)
		{
			this.Body = Body;
			this.StatusCode = StatusCode;
			this.Headers = Headers;
		}
	}
}

File: runtimes/java-21.0/src/main/java/io/openruntimes/java/RuntimeOutput.java
package io.openruntimes.java;

import java.util.Map;

public class RuntimeOutput {
    private final String body;
    private final int statusCode;
    private final Map<String, String> headers;

    public RuntimeOutput(
            String body,
            int statusCode,
            Map<String, String> headers
    ) {
        this.body = body;
        this.statusCode = statusCode;
        this.headers = headers;
    }

    public String getBody() {
        return this.body;
    }

    public int getStatusCode() {
        return this.statusCode;
    }

    public Map<String, String> getHeaders() {
        return this.headers;
    }
}
